README.md

```diff
@@ -5,69 +5,11 @@
 [![Circle CI](https://circleci.com/gh/weaveworks/cortex/tree/master.svg?style=shield)](https://circleci.com/gh/weaveworks/cortex/tree/master)
 [![Slack Status](https://slack.weave.works/badge.svg)](https://slack.weave.works)
 
-*NB this is a pre-beta service. Availability is not 100%. APIs might change. Data could be lost.*
-
-Cortex is an API compatible [Prometheus](https://prometheus.io) implementation, that natively supports multitenancy and horizontal scale-out clustering.
-
 Cortex is a [Weaveworks](https://weave.works) project that forms the monitoring backend of [Weave Cloud](https://cloud.weave.works).
 
+It provides horizontally scalable, long term storage for [Prometheus](https://prometheus.io) metrics when used as a [remote write](https://prometheus.io/docs/operating/configuration/#remote_write) destination.
```

Comment:
Reading this makes me wonder how I'd get the data out of it again; I think it's important to mention that Cortex also provides a horizontally scalable, Prom-compatible query API.